runtime/src/rpc.rs

```diff
@@ -35,6 +36,15 @@ const BLOCK_WAIT_TIMEOUT: Duration = Duration::from_secs(6);
 // number of storage entries to fetch at a time
 const DEFAULT_PAGE_SIZE: u32 = 10;
 
+// TODO: Convert to a utility function
+/// Keys in storage maps are prefixed by two `twox_128` hashes: the pallet name and the
+/// storage item names. Then, assuming the map uses the `Blake2_128Concat` hasher, the layout
+/// looks as follows:
+/// `twox_128("PalletName") ++ twox_128("ItemName") ++ Blake2_128Concat(key) ++ key`
```

Comment:
```suggestion
/// `twox_128("PalletName") ++ twox_128("ItemName") ++ Blake2(key) ++ key`
```
Nit: I think `Blake2_128Concat` means `Blake2(key) ++ key` so `key` would be in there twice

Reply:
Turns out that  `AssetRegistry::Metadata` is hashed with `Twox64Concat` :(

Reply:
Added a separate decoding function for that hasher